x-pack/plugins/cloud_security_posture/public/components/translations.ts

```diff
@@ -40,3 +40,33 @@ export const CSP_EVALUATION_BADGE_PASSED = i18n.translate(
     defaultMessage: 'PASSED',
   }
 );
+
+export const ERROR_LOADING_DATA = i18n.translate('xpack.csp.pageTemplate.loadErrorMessage', {
```

Comment:
general comment, we can still use `translations.ts` files. but we should probably return `<FormattedMessage/>` instead of `i18n.translate()` when we call them in tsx files